common/network-common/src/main/java/org/apache/spark/network/server/ChunkFetchRequestHandler.java

```diff
@@ -90,7 +90,6 @@ protected void channelRead0(
     ManagedBuffer buf;
     try {
       streamManager.checkAuthorization(client, msg.streamChunkId.streamId);
-      streamManager.registerChannel(channel, msg.streamChunkId.streamId);
```

Comment:
I think if you remove this, you should add it back to `NettyBlockRpcServer` for non external shuffle service. Otherwise, you'll introduce the bug you fixed with pr to the non external shuffle service.

Reply:
Good catch! Added it back to `NettyBlockRpcServer`.